platform/mellanox/iproute2/scripts/get_sdk_patches.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/python3
```

Comment:
Please try to add logs and redirect to logfile as here so it would help in case of any build issues 
``` https://github.com/vivekrnv/sonic-buildimage/pull/44 ```

Reply:
I have removed the script completely.